Workbooks/Performance/Single Page Application Insights/Single Page Application Website Insights.workbook

```diff
@@ -0,0 +1,666 @@
+{
+  "version": "Notebook/1.0",
+  "isLocked": false,
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Initial Load Performance Insights of Single Page Applications\r\nThis Workbook template allows you to view your Single Page Application (SPA) performance data logged by the <a href=\"https://github.com/Microsoft/Mezzurite\">Mezzurite Timing Library</a> and the <a href=\"https://github.com/Microsoft/ApplicationInsights-JS-Mezzurite\">Application Insights Mezzurite Plugin</a>\r\n\r\n"
+      },
+      "conditionalVisibility": null
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "query": "",
+        "crossComponentResources": [],
+        "parameters": [
+          {
+            "id": "e20e78fc-f0d0-40ba-b183-d68b351f91c3",
+            "version": "KqlParameterItem/1.0",
+            "name": "has_Mezzurite_Data",
+            "type": 1,
+            "isRequired": false,
+            "query": "customEvents\r\n| filter name == \"mz\"\r\n| where timestamp > ago(90d)\r\n| summarize value = tostring(sign(count()))\r\n\r\n\r\n\r\n\r\n\r\n",
```

Comment:
do you want to add a time range parameter at the top?  or you ALWAYS want this to be 90 days?

Reply:
This one we always want it to be 90 days as it looking back to see if you have mezzurite data.  The rest of the charts use a user set time range.

Reply:
Does since it looking at 90 days does it hit the slow path?  I could drop it to 30 days if that will be less intensive.